<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LedPanel</class>
 <widget class="QWidget" name="LedPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>109</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0" alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="ledRed">
     <property name="maximumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="led_widget_res.qrc">:/main/LED_Red_Off.png</pixmap>
     </property>
    </widget>
   </item>
   <item row="0" column="1" alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="ledYellow">
     <property name="maximumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="led_widget_res.qrc">:/main/LED_Yellow_Off.png</pixmap>
     </property>
    </widget>
   </item>
   <item row="0" column="2" alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="ledGreen">
     <property name="maximumSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="led_widget_res.qrc">:/main/LED_Green_Off.png</pixmap>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="led_widget_res.qrc"/>
 </resources>
 <connections/>
</ui>
